lib/importers/licence_transaction/facet_tagger.rb

```diff
@@ -0,0 +1,40 @@
+require "uri"
```

Comment:
This commit could be squashed into the previous one but not a blocker 👍

Reply:
This class doesn't exist in the previous commit ☺️